rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json

```diff
@@ -89,18 +89,21 @@
             "description": "Language value.",
             "type": "string",
             "enum": [
-              "fre",
```

Comment:
why did you change the list of languages? If it is really the case, you need to change also the form option.

Reply:
As discussed, adding a note to the commit message